src/containerapp/azext_containerapp/commands.py

```diff
@@ -50,52 +50,56 @@ def load_command_table(self, _):
     with self.command_group('containerapp env dapr-component') as g:
         g.custom_command('init', 'init_dapr_components', is_preview=True)
 
-    with self.command_group('containerapp service', deprecate_info=self.deprecate(redirect='containerapp add-on', hide=True), is_preview=True) as g:
+    with self.command_group('containerapp service', deprecate_info=self.deprecate(redirect='containerapp add-on', expiration='2.56.1', hide=True), is_preview=True) as g:
```

Comment:
Can you show me a picture about what it will looks like after added `expiration='2.56.1'`?

Reply:
![image](https://github.com/Azure/azure-cli-extensions/assets/85650284/010d3a57-69d3-4fd1-a860-ef99d88fb922)

Reply:
@bgashirabake The command group `containerapp service` only exist in azure-cli-extension, the expiration version should be extension version.

Reply:
Please note that after adding `expiration='2.56.1'`, if the customer is using a version greater than `2.56.1` , these commands will not work and cause breaking change to the customer's automation scripts

Reply:
They should use add-on instead

Reply:
Hi @bgashirabake 

Please note the breaking change on the `history.rst ` about if the customer is using the Azure CLI version greater than 2.56.1 , these commands will not work and cause breaking change.
For example:
```
* [Breaking Change]  'az containerapp service': deprecate from Azure CLI version 2.56.1
* 'az containerapp add-on' : support for add-on weaviate create and delete commands
```

Reply:
> Please note that after adding expiration='2.56.1', if the customer is using the Azure CLI version greater than 2.56.1 , these commands will not work and cause breaking change to the customer's automation scripts

@bgashirabake Have you evaluated the impact of this breaking change on customers? If it has almost no impact on customers, then I think this is acceptable

Reply:
> expiration='2.56.1'

Additionally, we have not yet released version `2.56.1`, the latest CLI version is ``2.56.0`

Reply:
@zhoxing-ms yes, we had slated the cut off time for the service command group to be the first release of 2024. Given that i don't know when the 2.56.1 cli version is coming out, i wanted to make it the last version where the service command group is usable. 
About impact: yes, the service group has had a deprecation warning for ~2 months now so we expect customers who use it to be aware. That + the whole time before 2.56.2 gets released seemed reasonable. Moreover, our docs use have used both add-on and service for some time now: https://learn.microsoft.com/en-us/cli/azure/containerapp?view=azure-cli-latest

Reply:
Hi @bgashirabake @zhoxing-ms 

The version of `2.56.1` is not exist.
The first release of 2024 is `2.56.0`: https://github.com/Azure/azure-cli/milestone/137
The next release is `2.57.0`: https://github.com/Azure/azure-cli/milestone/138